### ISSUE TITLE
ADS1299 connections to subject

### ISSUE THREAD

**Issue by wjcroft** [Ildaron/EEGwithRaspberryPI#5]
Hi,

If you look carefully at the TI ADS1299 datasheet material, the 'Bias' electrode is the SAME as Ground in other EEG systems. This means that a SEPARATE ground connection to the subject is NOT required. 

In fact if you ARE connecting the AGND or DGND to the subject, you are defeating the 1 gigaohm amplifier input impedance. Since the power supply ground connections are LOW impedance. And the ADS1299 channel pins, SRBx reference pin(s) and Bias are all high impedance connections, 1 gigaohm nominal. The higher the amplifier input impedance, the more immunity the amplifier has to skin impedance effects, and the lower the noise levels.

In some EEG applications, such as with electrode caps, it is common practice to place the Ground / Bias electrode on the midline, such as AFz, POz, CPz, etc. This applies the counter-mains injections evenly to both sides of the scalp.

However it is also common to see examples of Bias and Reference just being applied to opposite ear lobes. But in no cases should the Bias be applied OFF the head, such as to a hand (as shown in your original diagram). This is a poor location for anti-mains injection.

----

Additionally, it's highly recommended to BATTERY power your Raspberry PI, to eliminate any potential danger of mains voltages to find a path to the subjects head (by cheap power supply adapter malfunction). This could be done with an appropriate power bank. Or it is also possible to find medical grade wall power supplies. The medical certification number is 60601.

https://www.google.com/search?q=medical+5+volt+power+adapter+60601

William Croft

**Comment by Ildaron** [comment]
Thank you, of course for power supply  only battery 5V

**Comment by david-nuream** [comment]
Yes, you are assuming no fault in the power supply. That's the point from the OP. If you have a non-60601 power supply, they have not been designed/tested against such failures. And even if you have a 60601 certified supply, they can still fail due to users doing stupid things. The point is that while NORMALLY one would expect only 5V and limited Amperage, the fault case is a dead short across the power supply to the board, which then shorts out the board to connect the mains power to the electrode on your body. 

Yes, many things have to "work" for this to be a risk, but it's a risk.

**Comment by Ildaron** [comment]
> Yes, you are assuming no fault in the power supply. That's the point from the OP. If you have a non-60601 power supply, they have not been designed/tested against such failures. And even if you have a 60601 certified supply, they can still fail due to users doing stupid things. The point is that while NORMALLY one would expect only 5V and limited Amperage, the fault case is a dead short across the power supply to the board, which then shorts out the board to connect the mains power to the electrode on your body.
> 
> Yes, many things have to "work" for this to be a risk, but it's a risk.

thank you for your message, did you see ?
https://pieeg.com/liability-pieeg/

**Comment by david-nuream** [comment]
Sure, but that's why it's important for anyone reading this to understand the issues so they can do what's right for their situation.



Sent from my phone with my thumbs...
<https://aka.ms/AAb9ysg>
________________________________
